.travis.yml

```diff
@@ -17,6 +17,9 @@ before_install:
   - sudo echo "deb https://apt.repos.intel.com/openvino/2020 all main" | sudo tee - a /etc/apt/sources.list.d/intel-openvino-2020.list
   - sudo apt-get update
   - sudo apt-get install -y --no-install-recommends intel-openvino-dev-ubuntu16-2020.2.130
+  - sudo apt-get install -y megatools python3-pip
+  - python3 -m pip install --upgrade pip
+  - python3 -m pip install -r modules/8_reinforcement_learning/requirements.txt
```

Comment:
Please move to top and remove `python3-numpy` because numpy from `apt-get` will conflict with `numpy` from pip

Reply:
Done. BTW shouldn't `sudo apt-get update` be added before  `sudo apt-get install`.

Reply:
I believe that it's already called by Travis once for Ubuntu image.